h/static/scripts/directive/sort-dropdown.js

```diff
@@ -0,0 +1,11 @@
+module.exports = function () {
+  return {
+    restrict: 'E',
+    scope: {
+      sortBy: '=',
+      sortOptions: '=',
+      onChangeSortBy: '&',
```

Comment:
Is this necessary? `=` is a two-way binding, so shouldn't you simply be able to assign to `sortBy` within the directive?